.github/workflows/build.yaml

```
@@ -52,7 +52,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', 'pypy-3.7', 'pypy-3.8', 'pypy-3.9']
+        python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.8', 'pypy-3.9']
```

```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.9', 'pypy-3.10']
```

It seems that officially, pypy is on 3.9 and 3.10: https://www.pypy.org/download_advanced.html

We could also decide for pypy to leave 3.8 and add 3.10 (3.8, 3.9, 3.10).

---

Thanks, I noticed that. I wanted to put this into separate PR to make PR and commit history clean.